### ISSUE TITLE
Missing 2023 German Ground Crew Voices

### ISSUE THREAD

**Issue by KlebonasKoncius** [GaijinEntertainment/fmod_studio_warthunder_for_modders#239]
I don't see any German ground crew voices under "dialogs_wt_tanks_2023" that are present in the game currently; can you guys update the repository with the missing files?

**Comment by Uchuujin01** [comment]
Yes, right after we add in game edited, fixed version of German ground crew VO.
